integrationtests/iscsi_ps_scripts.go

```diff
@@ -30,12 +30,12 @@ Set-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\iSCSI Target" -Name AllowLoopBa
 Restart-Service WinTarget
 `
 
-type IscsiSetupConfig struct {
+type iSCSISetupConfig struct {
```

Comment:
I think this is different in Go, if it starts with uppercase then it's not exported in the package, with this change this becomes private to the package only, is this intended and ok?

Reply:
I checked that this is just for the tests so I think it's ok to make it private (even better than what it's now)